analysis.md

```diff
@@ -87,11 +87,11 @@ scripts:
   <table class="unstyled">
     <tr>
       <td></td>
-      <td style="width: 100%; text-align: center;">Contents</td>
+      <td style="width: 50vh; text-align: center;">Contents</td>
```

Comment:
I think this was meant to be `vw`?

Reply:
Completely forgot to check for other sizes! I have a fix in mind 👍